### ISSUE TITLE
About The Download Error.

### ISSUE THREAD

**Issue by Mr-WhiteMan** [FaisalUmair/udemy-downloader-gui#597]
hi everyone, download crashing because of the docx and pdf attachment thats why skip the attachments than download will not crash.... than ıf u want a attachments u must download manual form the course.

**Comment by jlarini** [comment]
It's correct, but...
unhappilly some videos (most of them) isn't correctly downloaded, it's sized 3k~6k... (Udeler says it's downloaded ok)
I've tried a couple of times. I got only 20% actually correct courses, other 80% becames "crashed"...

It's almost the same problem which happened about a month ago...

**Comment by vijay6696** [comment]
I have skipped the attachment option in the settings but still it is not getting past the the "Building the course data", any suggestions to this issue?

**Comment by Mr-WhiteMan** [comment]
> It's correct, but...
> unhappilly some videos (most of them) isn't correctly downloaded, it's sized 3k~6k... (Udeler says it's downloaded ok)
> I've tried a couple of times. I got only 20% actually correct courses, other 80% becames "crashed"...
> 
> It's almost the same problem which happened about a month ago...

Did u try to skip attachments? I had same problem too.

**Comment by Mr-WhiteMan** [comment]
> I have skipped the attachment option in the settings but still it is not getting past the the "Building the course data", any suggestions to this issue?

Did u save the settings after skip attachments ? I dıdnt have that problem,but I spend a lot of time to solve the download problem.

**Comment by vijay6696** [comment]
Yes I have saved the settings but still facing the same issue do u  have any idea how to solve this? please help me

![image](https://user-images.githubusercontent.com/84227463/118372353-d49ecf00-b5ce-11eb-8a3c-2203364c891d.png)


![image](https://user-images.githubusercontent.com/84227463/118372363-e84a3580-b5ce-11eb-9e66-aedadc30af92.png)

**Comment by jlarini** [comment]
> 
> 
> > It's correct, but...
> > unhappilly some videos (most of them) isn't correctly downloaded, it's sized 3k~6k... (Udeler says it's downloaded ok)
> > I've tried a couple of times. I got only 20% actually correct courses, other 80% becames "crashed"...
> > It's almost the same problem which happened about a month ago...
> 
> Did u try to skip attachments? I had same problem too.

yes, I tried skip attachments, skip subtitles, skip both... auto retry... same result... 
![image](https://user-images.githubusercontent.com/41792441/118374363-15c4be00-b592-11eb-8567-19e2f8933c9e.png)
as a said, it's happening for some days, but I guess it, yesterday...
there's some downloaded correctly and some not. see image

**Comment by ghost** [comment]
I'm having the same thing where the download keeps attempting to download a particular attachment.  if i skip attachment, then download is fine.  Is it possible to easily fix this bug by skipping the attachment after 3x failed tries and move on with the rest of the download?  I mention this because sometimes, the content creators simply remove an attachment.  This should be a quick and easy fix.

**Comment by ScrubDonkey** [comment]
Has nothing to do with attachments or anything. Almost all Udemy videos are secure now. It's over....

**Comment by Mr-WhiteMan** [comment]
> Eklerle veya herhangi bir şeyle ilgisi yok. Hemen hemen tüm Udemy videoları artık güvende. Bitti....

Please do not give false information, video download still continues on udemy.

**Comment by vijay6696** [comment]
> > Eklerle veya herhangi bir şeyle ilgisi yok. Hemen hemen tüm Udemy videoları artık güvende. Bitti....
> 
> Please do not give false information, video download still continues on udemy.

no it is not working for  me
